src/pages-and-resources/pages/PageCard.jsx

```diff
@@ -55,19 +50,14 @@ function PageCard({
               onClick={() => history.push(`${pagesAndResourcesPath}/${page.id}/settings`)}
             />
           )}
-        </Card.Title>
-
-        <div className="mb-2"><StatusBadge status={page.enabled} /></div>
+          </Card.Title>
+          <StatusBadge status={page.enabled} />
+        </div>
 
-        <Card.Text className="flex-grow-1 m-0">
+        <Card.Text className="m-0">
           {page.description}
         </Card.Text>
 
-        {(page.allowedOperations.enable && !page.enabled) && (
-          <Button variant="outline-primary" size="sm" onClick={handleClick} className="align-self-end">
-            {intl.formatMessage(messages['enable.button'])}
```

Comment:
whenever you remove something always make sure to remove it's related message as well